server.js

```diff
@@ -1,30 +1,104 @@
-import express from "express";
-import cors from "cors";
+import express from "express"
+import cors from "cors"
+import books from "./data/books.json"
 
 // If you're using one of our datasets, uncomment the appropriate import below
 // to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
+// import avocadoSalesData from "./data/avocado-sales.json"
+// import booksData from "./data/books.json"
+// import goldenGlobesData from "./data/golden-globes.json"
+// import netflixData from "./data/netflix-titles.json"
+// import topMusicData from "./data/top-music.json"
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
+const port = process.env.PORT || 8080
+const app = express()
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
+app.use(cors())
+app.use(express.json())
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
-});
+app.get("/", (request, response) => {
+  response.json({responseMessage: "Hi there! Type in /books to browse all books, or try /random-book to get a random book recommendation!"})
+})
+
+app.get("/books", (request, response) => {
```

Comment:
This part I don't really understand, do you use this code? Tried to search for a language code but didn't get it to work

Reply:
Try to use a url using this format, it should work

`https://project-express-api-6jhpdpxubq-lz.a.run.app/books?language_code=eng`
or
`https://project-express-api-6jhpdpxubq-lz.a.run.app/books?language_code=en-US`